omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bootstrap_validator.rb

```diff
@@ -75,10 +75,11 @@ def run!
   # backend components, allow data bootstrapping to be bypassed when
   # no chef-server-running.json is present but a secrets file is present.
   def bypass_bootstrap?
-    first_run? && secrets_exists?
+    first_run? && secrets_exists? && PrivateChef['postgresql']['external']
```

Comment:
Alright, it looks like that attribute has already landed. If you could change this to `PrivateChef['use_chef_backend']` instead of postgresql/external, that will do it.